packages/ui/src/data/navbar-menu-data.ts

```diff
@@ -1,57 +1,61 @@
 import { MenuGroupType, MenuGroupTypes } from '@components/navbar/types'
 
-export const navbarMenuData: MenuGroupType[] = [
+import { i18nextViewsInstance } from '../i18n/i18n'
+
+const { t } = i18nextViewsInstance
```

Comment:
not a blocker, but let's try passing this instance in as a prop to the components from `gitness` app.